compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1440,7 +1453,9 @@ static LogicalResult setTransposeLikeOpRootConfig(
   TileSizesListType tileSizes;
   tileSizes.push_back(flowTileSizes);
   tileSizes.push_back(parallelTileSizes);
-  tileSizes.push_back(/*reduction tile sizes=*/{});
+  // No need for tiling reduction dims and inner parallel dims.
+  tileSizes.emplace_back(parallelTileSizes.size(), 0);
```

Comment:
This line does seem to match the comment above. Is this supposed to be `parallelTileSizes`?